### PR TITLE
Add tests to 'bit' module

### PR DESCRIPTION
**Pull request by goswami-rahul**
- added tests to files of `bit/` module
- wrote alternative method for `bit/find_missing_number.py`
- `bit/subset.py` now returns `Set[tuples]` rather than `List[lists]`.

### CODE REVIEW

File: bit/subsets.py [hunk 1]
```diff
@@ -7,44 +7,54 @@
...
-def subnets(nums):
-    nums.sort()
-    total = 2 ** len(nums)  # or 1 << len(nums)
-    res = [] * total
+def subsets(nums):
+    """
+    :param nums: List[int]
+    :return: Set[tuple]
+    """
+    # nums.sort()  # no need for sort
```
Comment (christianbender) [inline bit/subsets.py]: Can we don't remove this line? You commented it out.

File: README_CN.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ python版数据结构和算法实现的简约版小示例
     - [single_number2：寻找出现1次的数（2)](bit/single_number2.py)
     - [single_number:寻找出现1次的数（1)](bit/single_number.py)
     - [subsets: 求所有子集](bit/subsets.py)
-    - [add_without_operator：无操作符的加法](bit/add_without_operator.py)
+    - [add_bitwise_operator：无操作符的加法](bit/add_without_operator.py)
```
Comment (christianbender) [inline README_CN.md]: What is with 无操作符的加法 ?

File: bit/count_ones.py [hunk 3]
```diff
@@ -14,16 +14,58 @@
...
+
+def count_ones_recur(n):
     """Using Brian Kernighan’s Algorithm. (Recursive Approach)"""
-    if not n: return 0
-    return 1 + count_ones(n & (n-1))
 
-def count_ones(n):
+    if not n:
+        return 0
+    return 1 + count_ones_recur(n & (n-1))
```
Comment (christianbender) [inline bit/count_ones.py]: I think it must be ```return 1 + count_ones(n & (n-1))``` because you renamed the function.

File: README_CN.md (hunk 2 above)
Comment (goswami-rahul) [reply README_CN.md]: This is the Chinese version of README of this repo. Don't know if it is still maintained, maybe @keon can tell about this.

File: bit/count_ones.py (hunk 3 above)
Comment (goswami-rahul) [reply bit/count_ones.py]: `return 1 + count_ones(n & (n-1))` was the old one. Now recursive version is renamed to `count_ones_recur`, and hence `return 1 + count_ones_recur(n & (n-1))`

File: bit/subsets.py (hunk 1 above)
Comment (goswami-rahul) [reply bit/subsets.py]: Yes, you are right. I will remove this.

File: bit/count_ones.py (hunk 3 above)
Comment (christianbender) [reply bit/count_ones.py]: @goswami-rahul  OK.

**Comment by christianbender** [comment]
@goswami-rahul I merge it.

**Pull request merged**